src/main/java/net/logstash/logback/appender/AbstractLogstashTcpSocketAppender.java

```diff
@@ -80,6 +82,11 @@
     public static final int DEFAULT_RECONNECTION_DELAY = 30000;
 
     /**
+     * The default reconnection to primary server delay (60000 milliseconds or 60 seconds)
+     */
+    public static final int DEFAULT_RECONNECTION_PRIMARY_DELAY = 60000;
```

Comment:
Hmm.  I think the default should be to not attempt to re-establish the connection.